lib/Model.js

```diff
@@ -85,7 +87,12 @@ class Model {
     /** NPM 6 */
 
     if (parsedOutput.advisories) {
-      Object.values(parsedOutput.advisories).forEach((a) => this.process(a));
+      Object.values(parsedOutput.advisories).forEach((a) => {
+        console.log(a.url.split("/")[4]);
```

Comment:
Was this `console.log` intentionally kept here? Just wondering!

Reply:
🤦🏻 Of course one debugging output is always missed 😆